connectivity/drivers/ble/FEATURE_BLE/TARGET_Ambiq_Micro/CMakelists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(mbed-os-ble-apollo3 INTERFACE)
+
+mbed_add_cmake_directory_if_labels("TARGET")
```

Comment:
Should this be:
```suggestion
if("Apollo3" IN_LIST MBED_TARGET_LABELS)
    add_subdirectory(TARGET_Apollo3)
endif()
```

Reply:
Yes, will need to refactor all of these.